Makefile

```diff
@@ -49,9 +62,114 @@ html-live: prep-docs
 		--port=0 \
 		$(SPHINXOPTS)
 
+# full build, no gallery
+# will not remove existing gallery files
 html-noplot: clean prep-docs
 	NAPARI_APPLICATION_IPY_INTERACTIVE=0 sphinx-build -M html docs/ docs/_build -D plot_gallery=0 -D sphinx_gallery_conf.examples_dirs=$(GALLERY_PATH) $(SPHINXOPTS)
 
+# just napari/docs
+# no generation from prep_docs scripts, no gallery
+# will not remove existing gallery files
+docs: clean prep-stubs
+	NAPARI_APPLICATION_IPY_INTERACTIVE=0 sphinx-build -M html docs/ docs/_build -D plot_gallery=0 -D sphinx_gallery_conf.examples_dirs=$(GALLERY_PATH) $(SPHINXOPTS) 
```

Comment:
(1) is `sphinx_gallery_conf.examples_dirs` needed with `plot_gallery=0`?
(2) I don't understand now how to build *both* docs and gallery? ie what is the incantation for a "full build" now?
(3) it might be worth mentioning that in the comment. "For gallery, look at XYZ. For a single gallery example, look at ABC. For a full build, look at ZYX."

Reply:
@jni 
1. my understanding from the docs is that examples_dir is a needed config, while plot_gallery is an execution option:
https://sphinx-gallery.github.io/stable/configuration.html
The settings are the same as existing make commands that didn't build gallery.

2. this hasn't changed. it's still make html.
3. Do you mean add comments to the existing make options? I can do that. I will definitely update the contribution guide before this is merged, just wanted some feedback on the options.

Reply:
@jni I update the contribution guide for the new make variants. I tried to make it clear which you'd want to run depending on what you were trying to contribute.